tests/integration_tests/allennlp_tests/test_allennlp.py

```diff
@@ -23,6 +23,52 @@ def test__set_param() -> None:
     assert params["model"]["hidden_size"] == [100, 200, 300]
 
 
+def test__build_params_overwiting_environment_variable() -> None:
```

Comment:
Ditto and typo.
```suggestion
def test_build_params_overwriting_environment_variable() -> None:
```

Reply:
Thank you for the catch, I fixed the typo. :pray:
I removed a consecutive underscore after confirming the question at https://github.com/optuna/optuna/pull/1449#discussion_r447619368.